component/src/main/java/org/wso2/extension/siddhi/store/rdbms/RDBMSConditionVisitor.java

```diff
@@ -341,7 +335,7 @@ public void endVisitStreamVariable(String id, String streamId, String attributeN
 
     @Override
     public void beginVisitStoreVariable(String storeId, String attributeName, Attribute.Type type) {
-        condition.append(this.tableName).append(".").append(attributeName).append(RDBMSTableConstants.WHITESPACE);
```

Comment:
Why the table name is removed ?

Reply:
Removed it to fix an issue which does not occur now. Added it back in 6f1396f